### PR TITLE
5272 eligibility permission group

### PR DESCRIPTION
**Pull request by weskubo-cgi**
Added all permissions associated with the Eligibility SPG/role.

### CODE REVIEW

File: backend/src/main/java/ca/bc/gov/hlth/hnweb/security/SecurityConfig.java [hunk 1]
```diff
@@ -39,6 +39,7 @@ protected void configure(HttpSecurity http) throws Exception {
             .mvcMatchers(HttpMethod.GET, "/docs/**").permitAll()
             .mvcMatchers(HttpMethod.POST,"/eligibility/check-msp-coverage-status").hasRole("E45")
             .mvcMatchers(HttpMethod.POST,"/eligibility/check-eligibility").hasRole("R15")
+            .mvcMatchers(HttpMethod.POST,"/eligibility/inquire-phn").hasRole("R41")
```
Comment (daveb-hni) [inline backend/src/main/java/ca/bc/gov/hlth/hnweb/security/SecurityConfig.java:42]: Should we have constants or enum for all the v2 types?

**Review by daveb-hni — COMMENTED** [review]
Looks good, one minor comment

File: backend/src/main/java/ca/bc/gov/hlth/hnweb/security/SecurityConfig.java (hunk 1 above)
Comment (weskubo-cgi) [reply backend/src/main/java/ca/bc/gov/hlth/hnweb/security/SecurityConfig.java:42]: Possibly. But we may end up renaming them in the future. Also, with all the String matching happening in this class I didn't feel that moving everything to constants was worth the loss of simplicity.

Comment (daveb-hni) [reply backend/src/main/java/ca/bc/gov/hlth/hnweb/security/SecurityConfig.java:42]: Yeah, it looks clean the way it is.

**Review by daveb-hni — APPROVED** [review]
Changes look good.

**Pull request merged**